index.html

```diff
@@ -188,34 +235,26 @@ <h2 class="project">Multi-Post Stories</h2>
           </a>
           <div class="c1-container">
             <div>
-              <h2 class="project">Tonic</h2>
               <h2 class="project">Facebook 360</h2>
             </div>
             <div class="descrcontainer">
               <div>
-                <p class="client">CANOPY</p>
                 <p class="client">FACEBOOK</p>
               </div>
```

Comment:
Please ensure that the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, margins).

## Desktop:
Your Implementation | Figma Design
:---:|:---:
 
![your-desk](https://user-images.githubusercontent.com/10332499/152401586-a03ed9ea-63d2-4af8-9c08-3968736ed5ec.PNG) | ![figma_desk](https://user-images.githubusercontent.com/10332499/152401696-8cf5d9ea-17bf-4f6d-9695-67da56ee7020.PNG)
 
## Mobile:
Your Implementation | Figma Design
:---:|:---:
![your-design](https://user-images.githubusercontent.com/10332499/152402326-02a50d37-ede9-47e5-88b4-8f69adc11d08.PNG)  |  ![figma](https://user-images.githubusercontent.com/10332499/152402250-d361e41d-2a01-42a7-b2ae-06da7d8558da.PNG)